.circleci/config.yml

```diff
@@ -75,10 +75,10 @@ jobs:
             python manage.py check
             python manage.py makemigrations --check
       - run:
-          name: Black
+          name: Ruff
           command: |
             . .venv/bin/activate
-            black -l 80 --check .
+            pytest --ruff .
```

Comment:
If you're just linting the code this can be `ruff format .` rather than using `pytest`